app/views/works/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_for @work do |f| %>
+<%= f.label :category %>
+<%= f.select :category, ["Album", "Book", "Movie"] %>
```

Comment:
You hardcoded the list of categories here, but it might've been cool to find a rails form helper to populate a select with all unique categories that works have. ;) Just a thought.